package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@cjaas/widgets",
```

Comment:
but this package will not get published. these is no entry file/no way to import any widgets the way it is currently st up?

Reply:
In that case we just skip the name then?

Reply:
i don't know if we need a package.json here since the root is not the package